arangod/Agency/Inception.cpp

```diff
@@ -547,12 +546,12 @@ void Inception::run() {
 // @brief Graceful shutdown
 void Inception::beginShutdown() {
   Thread::beginShutdown();
-  CONDITION_LOCKER(guard, _cv);
-  guard.broadcast();
+  std::lock_guard guard{_cv.mutex};
+  _cv.cv.notify_all();
 }
 
 // @brief Let external routines, like Agent::gossip(), signal our condition
 void Inception::signalConditionVar() {
-  CONDITION_LOCKER(guard, _cv);
-  guard.broadcast();
+  std::lock_guard guard{_cv.mutex};
+  _cv.cv.notify_all();
 }
```

Comment:
Not sure but looks like signal/notify_one should be instead of broadcast/notify_all